.github/workflows/e2e-tests.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           echo "build-matrix=$BUILD_MATRIX" >> "$GITHUB_OUTPUT"
 
   build-projects:
-    name: "E2E: Build ${{ matrix.buildGroup }}"
+    name: 'E2E: Build ${{ matrix.buildGroup }}'
```

Comment:
Not an issue now since the PR is closed, but note that here single quotes would make the name a static one rather than using the var.

Reply:
I don't think that's true for the Actions yaml. 

https://github.com/Automattic/jetpack/actions/runs/12049883691 appears to have the correct names for the runs.

Reply:
Ah, interesting...thanks!